other/materials_designer/create_point_defect_in_bulk.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>materials_with_defect_at_site_id</p><p>materials_with_defect_at_position</p><p> </p><p>and titles accordingly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>OK</p>